src/app/core/components/archive-settings-dialog/archive-settings-dialog.component.ts

```diff
@@ -66,6 +68,12 @@ export class ArchiveSettingsDialogComponent implements OnInit {
         });
     }
     this.archive = this.account.getArchive();
+    this.api.archive.getMembers(this.archive).then((response) => {
+      const members = response.getAccountVOs();
+      this.payer = members.find(
+        (member) => member.accountId === this.archive.payerAccountId
+      );
+    });
```

Comment:
What is this part doing? (Just curious.)

Reply:
It finds the payer objects, finding the memeber with the same account id as the payer account id of the archive